src/python/pythonnoderesources.hpp

```diff
@@ -44,7 +48,91 @@ struct PythonNodeResources {
 
     PythonNodeResources(PythonBackend* pythonBackend);
     ~PythonNodeResources();
-    static Status createPythonNodeResources(std::shared_ptr<PythonNodeResources>& nodeResources, const ::mediapipe::CalculatorGraphConfig::Node& graphNode, PythonBackend* pythonBackend, std::string graphPath);
+    static void createOutputTagNameMapping(std::shared_ptr<PythonNodeResources>& nodeResources, const ::mediapipe::CalculatorGraphConfig::Node& graphNodeConfig) {
```

Comment:
Why did you move definition from Cpp to the header?

Reply:
It was windows compilation problem. I have found a fix. We need to change the resource definition from struct to class. Otherwise, static member functions with definition in cpp files are not visible.